### ISSUE TITLE
There's something wrong with Roboto-Regular.woff2

### ISSUE THREAD

**Issue by Kazade** [Dogfalo/materialize#2660]
When using the file bundled with Materialize, I was getting the following error from Firefox:

```
downloadable font: GSUB: too large substitute: 65535 (font-family: "Roboto" style:normal weight:normal stretch:normal src index:1) source: ...
```

I reconverted Roboto Regular.ttf to woff2 and replaced the file and now it's working fine. I've also seen other people mentioning this on Stack Overflow.

**Comment by Kazade** [comment]
Actually I lie, it's not fixed by regenerating the woff2 file, at least by using this tool: https://everythingfonts.com/ttf-to-woff2 

I've tried several different methods of conversion, and various downloads from the web and every one shows the same issue with Roboto Regular and Roboto Light...

**Comment by MoxxNull** [comment]
I get kind of the same error, when using a "™" with ALT+0153 . 
`downloadable font: GSUB: too large substitute: 65535 (font-family: "Roboto" style:normal weight:300 stretch:normal src index:1) source: http://localhost:64806/font/roboto/Roboto-Light.woff2 site.sass.css:5584:12
downloadable font: Layout: Failed to parse lookup subtable 0 (font-family: "Roboto" style:normal weight:300 stretch:normal src index:1) source: http://localhost:64806/font/roboto/Roboto-Light.woff2 site.sass.css:5584:12
downloadable font: Layout: Failed to parse subtable 0 (font-family: "Roboto" style:normal weight:300 stretch:normal src index:1) source: http://localhost:64806/font/roboto/Roboto-Light.woff2 site.sass.css:5584:12
downloadable font: Layout: Failed to parse lookup 0 (font-family: "Roboto" style:normal weight:300 stretch:normal src index:1) source: http://localhost:64806/font/roboto/Roboto-Light.woff2 site.sass.css:5584:12
downloadable font: GSUB: Failed to parse lookup list table, table discarded (font-family: "Roboto" style:normal weight:300 stretch:normal src index:1) source: http://localhost:64806/font/roboto/Roboto-Light.woff2`

**Comment by Dogfalo** [comment]
It's possible the font was updated by Google  to fix this. We'll look into updated it for next release.

**Comment by brandonparsons** [comment]
I'm seeing this error also. Can I ignore it until a new version of Materialize comes out, or is this a big problem? I have no idea what the error means :)

**Comment by NitroBAY** [comment]
still have bug

**Comment by Pfahlf** [comment]
Still have this problem

**Comment by Drulac** [comment]
Still have this problem

**Comment by keyscores** [comment]
@Dogfalo I'm on the latest meteor bundle. Firefox cannot load roboto at all, it's falling back to open sans.

Here are some logs:

```
downloadable font: rejected by sanitizer (font-family: "Roboto" style:normal weight:normal stretch:normal src index:1) source: http://localhost:3000/packages/materialize_materialize/fonts/roboto/Roboto-Regular.woff2 

downloadable font: rejected by sanitizer (font-family: "Roboto" style:normal weight:normal stretch:normal src index:2) source: http://localhost:3000/packages/materialize_materialize/fonts/roboto/Roboto-Regular.woff

downloadable font: rejected by sanitizer (font-family: "Roboto" style:normal weight:normal stretch:normal src index:3) source: http://localhost:3000/packages/materialize_materialize/fonts/roboto/Roboto-Regular.ttf
```

**Comment by keyscores** [comment]
What worked for me was adding this line to main.css

`@import url('https://fonts.googleapis.com/css?family=Roboto:100,400');
`
*Pay attention to the weights you need, like 100, 400 etc.

**Comment by lorduriel** [comment]
I´m still having this problem in my production server, but not in loca enviroment.
I tried the @keyscores  answer but still have this problem.
I'm using v 0.97.8

**Comment by Dogfalo** [comment]
Fixed in 5f8b46c7de5b8ce0b11e929a271495e1806b9eb0. Update Roboto files from upstream, should have proper hinting which was causing the warnings.
